requirements.txt

```diff
@@ -4,5 +4,5 @@ numpy>=1.16.4
 torch>=1.1
 torchvision>=0.4.0
 pandas>=0.24  # lower version do not support py3.7
-test-tube>=0.7.5
+tensorboard>=1.14
```

Comment:
Is this needed? I thought we were using `SummaryWriter` so that we don't need to pull in tensorboard.

Reply:
Nm, disregard this comment. Looks like it's needed.

Reply:
https://github.com/PyTorchLightning/pytorch-lightning/pull/687#issuecomment-574754384